data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -204,13 +115,10 @@ rm --force /etc/kubernetes/manifests/machineconfigoperator-bootstrap-pod.yaml
 echo "Starting cluster-bootstrap..."
 
 # shellcheck disable=SC2154
```

Comment:
@sttts can you add some comments here like
* what are the input directories that this `cluster-bootstrap` acts on.
* what are some exit criterias
* also add docs on how to troubleshoot errors from this command (maybe just pointing to docs in the repo `cluster-bootstrap`)

Reply:
on it.